### DISCUSSION TITLE
Questions about different energy values calculated from compressed .pb model and uncompressed .pb model

### DISCUSSION THREAD

**Discussion by PolyuWeldingSpock** [deepmodeling/deepmd-kit#2966] [Q&A] [answered]
Hi developers of Deepmd kit. I have trained a deep potential model with DPGEN workflow. Then I extracted the deep potential after several iterations.  The original deep potential model (frozen_model.pb) was not compressed. I have used `dp compress -i frozen_model -o frozen_model_compressed` to get a compressed model frozen_model_compressed.pb. However, I found that when I used this two potentials to make a minimize calculation in lammps, these two deep potentials gave different values. 

I want to know that which energy was correct? Might some bad issues happened in compress progress? 

I have put files containing conf.lmp, input.lammps, frozen_model.pb and frozen_model_compressed.pb in this link for your test. https://1drv.ms/u/s!AtO91yh6_pLhhtlUJxk9vOikxqgZTQ?e=KVt20B

**Comment by njzjz** [comment, marked as answer]
What is your version? Recently, we fixed a bug in #2924.

**Reply by PolyuWeldingSpock** [reply]
I have installed newest version 2.2.7 dev. Problem seems to be fixed. Thanks.
